File: go/raindrops/raindrops.go
package raindrops

import "strconv"

func Convert(num int) string  {
	convertedString := ""

	if num % 3 == 0 {
		convertedString += "Pling"
	}

	if num % 5 == 0 {
		convertedString += "Plang"
	}

	if num % 7 == 0 {
		convertedString += "Plong"
	}

	if convertedString == "" {
		convertedString = strconv.Itoa(num)
	}

	return convertedString
}

File: typescript/raindrops/raindrops.ts
class Raindrops {

    convert(num:number) :string{
        let out = ''
        if (num % 3 == 0) {out+="Pling"}
        if (num % 5 == 0) {out+="Plang"}
        if (num % 7 == 0) {out+="Plong"}
        if (out === '') {out = num.toString()}

        return out
    }
}

export default Raindrops
